src/NzbDrone.Api/Series/MovieModule.cs

```diff
@@ -140,6 +140,24 @@ private PagingResource<MovieResource> GetMoviePaged(PagingResource<MovieResource
 				pagingSpec.FilterExpression = v => v.MovieFileId == 0;
 			}
 
+            if (pagingResource.FilterKey == "title")
+            {
+                if (pagingResource.FilterType == "contains")
+                {
+                    if (pagingResource.FilterValue == string.Empty || pagingResource.FilterValue == null)
```

Comment:
i should pull this out of the contains...  i.e. the contains or not contains case should be within this case.. gonna fix and commit that now.. just a sec ;)